src/Features/Core/Portable/CodeFixes/Async/AbstractConvertToAsyncCodeFixProvider.cs

```diff
@@ -5,34 +5,66 @@
 #nullable disable
 
 using System;
+using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
 
 namespace Microsoft.CodeAnalysis.CodeFixes.Async
 {
-    internal abstract partial class AbstractChangeToAsyncCodeFixProvider : AbstractAsyncCodeFix
+    internal abstract partial class AbstractConvertToAsyncCodeFixProvider : CodeFixProvider
```

Comment:
there were two abstract base types.  the only change here is merging those two abstract base types into a single type.